api/application/applications.go

```diff
@@ -8,139 +8,114 @@ import (
 )
 
 //WIP
-func Fetch(network *net.Network) string {
+func Fetch(network *net.Network) (string, error) {
 	url := "/v1/applications/fetch"
 	body, err := network.SendGetRequest(url)
 	if err != nil {
-		fmt.Println(err)
+		return string(body), err
```

Comment:
Good catch.  As there are numerous places with this in the code can we just accept this and other such pull requests, and I will go through the codebase on a second pass when they are all merged into master, and fix these cases.